aries-site/src/examples/components/table/TableSingleSelectExample.js

```diff
@@ -193,27 +201,55 @@ const columns = [
   },
 ];
 
-const onClickHandler = record => {
-  // eslint-disable-next-line no-alert
-  alert(`
-    Record was clicked:
-    { 
-        id: ${record.id},
-        orderName: ${record.orderName}
-        state: ${record.state}
-        orderDate: ${record.orderDate}
-    }
-    
-    You can use onClick() to navigate to a record's detail
-    page, open a panel or modal to edit the record, or perform 
-    other actions as you see fit.
-  `);
-};
-
 export const TableSingleSelectExample = () => {
   const size = React.useContext(ResponsiveContext);
+  const [pageDetails, setPageDetals] = React.useState('');
 
-  return (
+  return pageDetails.id !== undefined ? (
+    <>
+      <Button
+        onClick={() => {
+          setPageDetals('');
+        }}
+        alignSelf="start"
+        icon={<FormPrevious />}
+        label="Orders"
+       />
+      <Box margin={{ horizontal: 'large' }} border="bottom">
+        <Heading level={3}>Order Number: {pageDetails.id}</Heading>
+      </Box>
+      <Text
+        margin={{ horizontal: 'large', top: 'large', bottom: 'medium' }}
+        size="xxlarge"
+      >
+        Details
+      </Text>
+      <Box
+        margin={{ horizontal: 'large' }}
+        gap="medium"
+        direction="row-responsive"
+      >
+        <Box gap="small" direction="column">
+          <Text>Order Name:</Text>
```

Comment:
All of these should be `size="small"`

Reply:
ok ill add that for the ones below as well